examples/dummysolver/main_dummysolver.cc

```diff
@@ -42,7 +42,7 @@ try {
         getParamFromGroup<std::string>("preCICE", "ConfigFileName");
     const std::string meshName =
         getParamFromGroup<std::string>("preCICE", "MeshName");
-    const precice::string_view meshNameView(meshName);
+    const std::string meshNameView(meshName);
```

Comment:
We don't need a separate view of strings anymore, do we? Should be able to replace all `meshNameView` etc. with the string versions.

Reply:
We could also just pass literals in every preCICE API call.